setting-up-otp-mfa.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Setting up OTP multi-factor authentication
```

Comment:
Food for thought: Is "OTP" how we want to define this kind of MFA? Perhaps "authenticator app" would be a more user friendly term

Reply:
I'm not sure. With WebAuthn we stuck with the standard name rather than the name browser vendors are promoting (Passkey). Parenthetical maybe? Ie "Setting up OTP (authenticator app) multi-factor authentication"?

Reply:
> With WebAuthn we stuck with the standard name rather than the name browser vendors are promoting (Passkey)

I don't think that's entirely true, I think we mainly call webauthn "security device" in the UI
<img width="706" alt="Screenshot 2023-03-29 at 2 14 34 PM" src="https://user-images.githubusercontent.com/42748004/228630504-87771926-9e2a-491a-a6c9-7c53788ae0dd.png">

Hmm, I like the parenthetical version except that it might be too long? I'm fine with keeping OTP, but is it conventional to call it OTP or TOTP?